services/transaction/tests/constants/multisignature.js

```diff
@@ -13,33 +13,56 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const transactions = [{
-	serviceId: '22139cb8-d1d3-43ec-b7d3-9f1ed30e5957',
-	nonce: 1,
-	senderPublicKey: '3e50549cd4d98760064ff2fe51801afba4e5e8623335275cece0eeff8495a81b',
-	senderAddress: 'lsk6f9gs9zfk6wqxgxnmxw5tbce2g8d42krtrwdfd',
-	asset: '{"numberOfSignatures":2,"mandatoryKeys":["228c865b903dab827342aa6611676bf883e982e7cd467c9168a7966cdabb391c","3e50549cd4d98760064ff2fe51801afba4e5e8623335275cece0eeff8495a81b"],"optionalKeys":[]}',
-	moduleAssetId: '4:0',
-	fee: '314000',
-	createdAt: 1629276090,
-	modifiedAt: 1629276090,
-	expiresAt: 1662224456,
-},
-{
-	serviceId: '088ea73f-cf58-475e-8fec-128be99555bf',
-	nonce: 2,
-	senderPublicKey: '4d50549cd4d98760064ff2fe51801afba4e5e8623335275cece0eeff8495a81c',
-	senderAddress: 'lsk2dp8gf6me3hafoqgtqej8dk96uusdhykvnkbrr',
-	asset: '{"numberOfSignatures":2,"mandatoryKeys":["4d50549cd4d98760064ff2fe51801afba4e5e8623335275cece0eeff8495a81c","9bc945f92141d5e11e97274c275d127dc7656dda5c8fcbf1df7d44827a732664"],"optionalKeys":[]}',
-	moduleAssetId: '4:0',
-	fee: '640000',
-	createdAt: 1629276090,
-	modifiedAt: 1629276090,
-	expiresAt: 1662224456,
-},
+const transactions = [
+	{
+		serviceId: '22139cb8-d1d3-43ec-b7d3-9f1ed30e5957',
```

Comment:
Is this format the new hash?

Reply:
No, this is UUIDv4 format. I'll update it.